i18n-country-translations.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.platform     = Gem::Platform::RUBY
 
-  s.add_dependency('i18n', '~> 1.0.1')
+  s.add_dependency('i18n', '>= 1.0.1', '< 2')
```

Comment:
Should we directly go to `">= 1.6.0"` then? I'm always puzzled how we should push people to update dependencies so we do not have to manage conflicts like this.

Reply:
Going `">= 1.6.0"` might paint us in a corner.

Anyway, the I18n API is so small that I'm not worried about compatibility issue between versions.